managed/services/backup/backup_service.go

```diff
@@ -264,12 +265,12 @@ func checkMongoBackupPreconditions(q *reform.Querier, service *models.Service, s
 }
 
 type prepareRestoreJobParams struct {
-	AgentID      string
-	ArtifactName string
-	DBVersion    string
-	Location     *models.BackupLocation
-	ServiceType  models.ServiceType
-	DBConfig     *models.DBConfig
+	AgentID       string
+	ArtifactName  string
+	DBVersion     string
+	LocationModel *models.BackupLocation
```

Comment:
Not sure why you changed name?

Reply:
We also have `models.BackupLocationConfig` which not a model. So, the new name makes it clear what kind of object we're working with.